local.env

```diff
@@ -0,0 +1,10 @@
+
+# ECSDA Account ID
+OPERATOR_ID_A=0.0.2
+
+# ECDSA DER Encoded Private Key
+OPERATOR_KEY_A= 302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137
+
+# ECDSA HEX Encoded Private Key
+HEX_PRIVATE_KEY_A= 0x2e1d968b041d84dd120a5860cee60cd83f9374ef527ca86996317ada3d0d03e7
```

Comment:
These are the ones straight from the local node docs,  right?  (I hope so!) Anyway, leave a comment in here stating that (a link to their readme too would be good to back it up).

Reply:
Huh, really? They had it up there? I got these configurations from the old `hardhat.config.js` file, and they were publicly available. That's why I thought it would be suitable for the `local.env` file

Reply:
well they probably _are_ publicly available - is it [one of these](http://json.schemastore.org/prettierrc)?  (Too lazy to look, sorry)